src/index.ts

```diff
@@ -245,6 +252,52 @@ async function deployService(serviceName: ServiceName, app: AppConfig) {
   ).pipeStdout!(process.stdout);
 }
 
+// dev: this method is not designed like I'd like to
+// Ideally, this method reads the route from the deployment wrangler config, and test it
+// However, this does not seem to be exposed by wrangler (cli/library), and doing a full parsing is too much much
+// Therefore, we provide a dedicated key test.hostname in the app config, and test that hostname
```

Comment:
how about parsing the config.yaml file. It's a local file and its format is not so complex as the wrangler one.